test/e2e/_www/liquid-articles/second_article.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
```

Comment:
e2e generated artifact should be not commited. Otherwise it is nice!
Can you fix that and adding a CHANGES entry? Thanks in advance!